firestore/query.go

```diff
@@ -162,10 +163,19 @@ func (q Query) Offset(n int) Query {
 	return q
 }
 
-// Limit returns a new Query that specifies the maximum number of results to return.
-// It must not be negative.
+// Limit returns a new Query that specifies the maximum number of first results
+// to return. It must not be negative.
 func (q Query) Limit(n int) Query {
 	q.limit = &wrappers.Int32Value{Value: trunc32(n)}
+	q.limitToLast = false
+	return q
+}
+
+// LimitToLast returns a new Query that specifies the maximum number of last
```

Comment:
Will this the meaning of this first / last results distinction be obvious to users of Firestore? @BenWhitehead